Sources/Client/Client/Client.swift

```diff
@@ -109,7 +94,7 @@ public final class Client {
     }
     
     /// Weak references to channels by cid.
-    let channelsAtomic = Atomic<[ChannelId: [WeakRef<Channel>]]>([:])
+    let watchingChannelsAtomic = Atomic<[ChannelId: [WeakRef<Channel>]]>([:])
```

Comment:
Why isn't this `[ChannelId: WeakRef<Channel>]` but `[ChannelId: [WeakRef<Channel>]]`? Can we have more than 1 channel with the same channelId?

Reply:
Yes, you're right we can have multiple channels with the same `cid`. For example, when you create a channel and then will get a populated channel from the response.